### ISSUE TITLE
MD038 disallows line breaks for inline code snippets

### ISSUE THREAD

**Issue by raxod502** [DavidAnson/markdownlint#130]
I commonly have text like this, as a result of hard line wrapping:

```
There are a few other Yarn tasks available, each runnable with `yarn
<task>`.
```

But it is disallowed by MD038; why? Is there something else I should be doing instead?

**Comment by DavidAnson** [comment]
This seems like a bug in my code, sorry!

https://spec.commonmark.org/0.28/#code-spans

**Comment by DavidAnson** [comment]
Could you please try your example here: https://dlaa.me/markdownlint/

What you have above does not trigger MD038 for me. Might there be something else to this?

**Comment by raxod502** [comment]
Sorry, I really should have made sure this was reproducible as a standalone example before opening this. It appears that a third line is necessary:

```markdown
# Heading

There are a few other Yarn tasks available, each runnable with `yarn
<task>`. The `dev` task actually just runs `server` and `watch` in
parallel. The `server` task serves the built static files, while
```

**Comment by DavidAnson** [comment]
No worries! It’s probably the odd count of ticks that’s confusing things. I’ll fix this soon, sorry again.
